deskpro/deskpro-admin-frontend
# Feature/9136 voice validation

Test steps:
- Go to http://localhost:3000/#/voice/settings
- Empty the `Ring Time` Units filed
- Input any string in the `Ring Time` Units filed
- Observe `Brand` selectbox in the bottom of page

Expected behavior:
- Must display error messages above cases
